src/common/components/header-with-content-link.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'm rethinking the name of this again. Could we change it to `HeadingWithContentLink` instead? I think that would clarify that it does not contain a `<header>` element.

Reply:
Ok